.travis.yml

```diff
@@ -3,7 +3,9 @@ cache: bundler
 language: ruby
 dist: trusty
 rvm:
-  - jruby-9.0.1.0
+  - jruby-9.0.5.0
+  - jruby-9.1.12.0
```

Comment:
Thanks for the update. Just a few hours ago JRuby 9.1.13.0 was released (!) .
http://jruby.org/2017/09/06/jruby-9-1-13-0

This is already available on Travis CI. Would please you update it?

Reply:
This release includes bug fixes for `Enumerable#each_with_index`.
jruby/jruby#4724.